src/demux/id3.ts

```diff
@@ -357,7 +358,11 @@ class ID3 {
       }
     }
     return out;
-  }
+  };
+
+  export const testables = {
```

Comment:
I like the call out that this export is only for testing :)

Reply:
I can't take [credit](https://stackoverflow.com/questions/54116070/how-to-unit-test-non-exported-functions) for that one, but yes it's a decent way to expose module internals ^^